app/js/questions_controller.js

```diff
@@ -7,8 +7,10 @@
   var MSG_NO_QUESTIONS = 'No questions found';
   var QUESTION_LIST_TMPL = 'question_list_day.html';
 
-  var question_list_container;
-  var load_more_button;
+  var my_questions_list_container;
```

Comment:
We should rename new_questions_list_container and basically rename all the my_questions references.